### ISSUE TITLE
Datepicker: Month Year picker doesnt have select button.

### ISSUE THREAD

**Issue by janahintal** [infor-design/enterprise#2557]
**Describe the bug**
Month Year picker doesnt have select button. Thus, the user has to click it twice to select both desired Month/year.

**To Reproduce**
1. Go to https://4200-beta1-enterprise.demo.design.infor.com/components/datepicker/example-index.html

**Expected behavior**
Should have submit button for the user has to double select to select both desired Month & Year

**Version**
Beta 4.20

**Screenshots**
NA

**Platform**
All devices, OS and Browser. 

**Additional context**
NA

**Comment by tmcconechy** [comment]
I reviewed this with design and they felt the same way.

So we have
1. http://master-enterprise.demo.design.infor.com/components/datepicker/example-timeformat.html
2. http://master-enterprise.demo.design.infor.com/components/datepicker/example-index.html
3. http://master-enterprise.demo.design.infor.com/components/datepicker/example-month-year-format.html

I think the only two changes are:
a) make clicking a day not close the dialog
b) on 2 make the select/cancel show on the month/year dropdown (picker) and that will close that dialog instead of clicking a month or day and go back to the “main” page

The only consequence of this is a slight behavior change that clicking a day will mean you now have to click select after to close, but especially when there was a time picker or month picker this was also mentioned in issues that the dialog closed right away. Enter will still close when hit (same is clicking submit)  

cc @nickwynja  and @clepore any feedback?

**Comment by tmcconechy** [comment]
Waiting on https://github.com/infor-design/enterprise/pull/2583 to be merged

**Comment by brianjuan** [comment]
Found a couple of issues.
Issue 1. On https://4200-rc2-enterprise.demo.design.infor.com/components/datepicker/example-index.html, Select button for the entire date picker does not work when you click it right after selecting month and year.
Steps:
1. Open date picker
2. Open Month and Year dropdown, select any month and year, then click Select button, dropdown will close
3. Now click the Select button for the entire date picker. See issue: nothing happens, current selected date will not be applied.

Issue 2: Month and Year Select button does not work in Range, https://4200-rc2-enterprise.demo.design.infor.com/components/datepicker/example-range.html#
Steps:
1. Open date picker in "Range - No Initial value"
2. Open Month and Year dropdown, select any month and year, click Select button, see issue: Month and year will not be applied.

**Comment by brianjuan** [comment]
Issues above are now fixed. 
However, on Calendar https://4200-rc3-enterprise.demo.design.infor.com/components/calendar/example-index.html and Month View https://4200-rc3-enterprise.demo.design.infor.com/components/monthview/example-index.html, there is no select button so there is no way to apply the selected Month and Year. I overlooked the testing for calendar and monthview yesterday, is this issue part of this ticket or should be in separate ticket?
![image](https://user-images.githubusercontent.com/39508864/62929482-26f4ff80-bded-11e9-8ef4-5c2d7014dd42.png)
